barretenberg/cpp/pil/avm/avm_mini.pil

```diff
@@ -72,23 +77,38 @@ namespace avmMini(256);
     rwb * (1 - rwb) = 0;
     rwc * (1 - rwc) = 0;
     
+    // TODO: Constrain rwa, rwb, rwc to u32 type and 0 <= in_tag <= 6
+
+    // Set intermediate registers to 0 whenever tag_err occurs
+    tag_err * ia = 0;
+    tag_err * ib = 0;
+    tag_err * ic = 0;
+
     // Relation for addition over the finite field
+    #[SUBOP_ADDITION_FF]
     sel_op_add * (ia + ib - ic) = 0;
     
     // Relation for subtraction over the finite field
+    #[SUBOP_SUBTRACTION_FF]
     sel_op_sub * (ia - ib - ic) = 0;
 
     // Relation for multiplication over the finite field
+    #[SUBOP_MULTIPLICATION_FF]
```

Comment:
why the subop prefix?

Reply:
Refers to the sub_operations table (also named main execution). I prefixed the relation identifiers to point to the relevant table (MEM for memory trace).

Should I rather prefix all with MAIN instead of SUBOP ?

Reply:
no no subop is good i just wasnt sure of its origin!